edenia/antelope-tools
# Feat/use subscription 1106

### Use subscription for update the health check status

### What does this PR do?

- Resolve #1106 
- Storage the supported APIs in the DB
- Show server version
- Adjust the UI of the BPs cards and use a high resolution image for empty state

### Steps to test

1. Run the project locally.
2. Go to "Nodes page" and "Endpoints page".
3. Check that every two minutes the info of the endpoint is updated and the page is not rerendered.

### Screenshots

#### **Endpoints Page** 
![imagen](https://user-images.githubusercontent.com/66583677/211870523-60849cf9-5919-4c4d-8c2e-7ae840862472.png)

#### **Nodes Page** 
![imagen](https://user-images.githubusercontent.com/66583677/211870709-f1649d25-47c4-4107-b837-a76d545602ab.png)

#### **BPs Page**
![imagen](https://user-images.githubusercontent.com/66583677/211871040-2b3123cc-4a95-4897-9e3d-e0295e54ccfe.png)
